snapcraft/storeapi/errors.py

```diff
@@ -357,6 +357,28 @@ def __init__(self, result):
         super().__init__()
 
 
+class StoreInternalError(StoreError):
+
+    fmt = '{what}: {why}\n{action}'
+
+    def __init__(self, response):
+        what = 'The store encountered an internal error'
+        why = None
```

Comment:
There will never be a case (and if there is, wee shouldn't show) any scenario where we have json for a 500. We should also leave this as a fixed message error, raised when we do not find json in a response.

Reply:
Alright, I've updated the PR to no longer use the error list.